src/schemas/json/function.json

```diff
@@ -514,6 +516,59 @@
           "description": "The text of the email."
         }
       }
+    },
+    "sqlBinding": {
+      "properties": {
+        "connectionStringSetting": {
+          "type": "string",
+          "description": "The name of the app setting that contains the SQL connection string used to connect to the database."
+        }
+      },
+      "oneOf": [
+        {
+          "properties": {
+            "type": {
+              "const": "sqlTrigger"
+            },
+            "tableName": {
+              "type": "string",
+              "description": "The name of the table to be monitored for changes."
+            }
+          }
+        },
+        {
+          "properties": {
+            "type": {
+              "const": "sql"
+            },
+            "direction": {
+              "const": "in"
+            },
+            "commandText": {
```

Comment:
At first, I could not understand that by `Command` we mean sql query. If it makes sense, please update the naming here. Some options could be `SqlQuery` and `SqlQueryType` or even `queryText` and `queryType` look good IMO.

Reply:
As with the others - this is something that would require modification on the extension side and would be a breaking change. I'll take up all these suggestions with the team and decide what we want to do but for purposes of this PR the schema is accurate with the current state of the extension as it is now.